Flaky Test: TestMetric10kDPS/TestMetricsFromFile

```
=== RUN   TestMetricsFromFile
    metric_test.go:[113](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/5046360176?check_suite_focus=true#step:14:113): 
        	Error Trace:	metric_test.go:113
        	Error:      	Received unexpected error:
        	            	unknown field "intGauge" in v1.Metric
        	Test:       	TestMetricsFromFile
--- FAIL: TestMetricsFromFile (0.00s)
```

Examples:
https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/5046360176?check_suite_focus=true#step:14:110
https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/5044466308?check_suite_focus=true#step:14:109
https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/5044517528?check_suite_focus=true#step:14:106

--- a/testbed/tests/testdata/k8s-metrics.json
+++ b/testbed/tests/testdata/k8s-metrics.json
@@ -45,11 +45,11 @@
                             "unit": "1",
                             "gauge": {
                                 "dataPoints": [
-                                   {
-                                      "asInt": "1",
-                                      "startTimeUnixNano": "1612819530934424200",
-                                      "timeUnixNano": "1612819531934424200"
-                                   }
+                                    {
+                                        "asInt": "1",
+                                        "startTimeUnixNano": "1612819530934424200",
+                                        "timeUnixNano": "1612819531934424200"
+                                    }
                                 ]
                              }
                         },
@@ -60,9 +60,9 @@
                             "gauge": {
                                 "dataPoints": [
                                     {
-                                       "asInt": "1",
-                                       "startTimeUnixNano": "1612819530934424200",
-                                       "timeUnixNano": "1612819531934424200"
+                                        "asInt": "1",
+                                        "startTimeUnixNano": "1612819530934424200",
+                                        "timeUnixNano": "1612819531934424200"
                                     }
                                  ]
                             }

--- a/testbed/tests/testdata/k8s-metrics.json
+++ b/testbed/tests/testdata/k8s-metrics.json
@@ -43,26 +43,28 @@
                             "name": "k8s.deployment.desired",
                             "description": "Number of desired pods in this deployment",
                             "unit": "1",
-                            "intGauge": {
+                            "gauge": {
                                 "dataPoints": [
-                                    {
-                                        "timeUnixNano": "1612819531934424200",
-                                        "value": "1"
-                                    }
+                                   {
+                                      "asInt": "1",
+                                      "startTimeUnixNano": "1612819530934424200",
+                                      "timeUnixNano": "1612819531934424200"
+                                   }
                                 ]
-                            }
+                             }
                         },
                         {
                             "name": "k8s.deployment.available",
                             "description": "Total number of available pods (ready for at least minReadySeconds) targeted by this deployment",
                             "unit": "1",
-                            "intGauge": {
+                            "gauge": {
                                 "dataPoints": [
                                     {
-                                        "timeUnixNano": "1612819531934424200",
-                                        "value": "1"
+                                       "asInt": "1",
+                                       "startTimeUnixNano": "1612819530934424200",
+                                       "timeUnixNano": "1612819531934424200"
                                     }
-                                ]
+                                 ]
                             }
                         }
                     ]

